deploy/agent/clusternet_agent_rbac.yaml

```diff
@@ -23,6 +23,34 @@ subjects:
     name: clusternet-agent
     namespace: clusternet-system
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
No need to define extra roles. ServiceAccount `clusternet-app-deployer` has already been granted with full access. Just reuse that token when listing pods/nodes.